src/main/java/tdl/s3/sync/Destination.java

```diff
@@ -41,10 +57,67 @@ public AWSSecretsProvider getSecret() {
         return secret;
     }
 
-    public AmazonS3 buildClient() {
-        return AmazonS3ClientBuilder.standard()
+    public AmazonS3 getClient() {
```

Comment:
At the end of this refactoring, there should be no need for getClient(), getBucket() and getPrefix()